### ISSUE TITLE
[Web UI] Failed to route/load Business Management View Pages

### ISSUE THREAD

**Issue by panpan0000** [tkestack/tke#1785]
Business Panel 404 (业务管理平台按钮跳转问题)
**What happened**:
![image](https://user-images.githubusercontent.com/14049268/156473620-6a7ceace-d37f-4697-b62e-c19937037cf2.png)

when clicking the button, it jumps to 
http://172.30.120.61/tkestack-project
then it leads to page 
![image](https://user-images.githubusercontent.com/14049268/156473666-9ded7d14-fc66-46ce-9a0d-fc18fd62ce6e.png)

```
对不起，加载页面时遇到了错误，[点击重试]()！
```

浏览器Console报错
```
index.project.main.0b86c842259010072178.js:2 Uncaught TypeError: Cannot read properties of undefined (reading 'Manager')
    at Object.71934 (index.project.main.0b86c842259010072178.js:2:1380728)
    at i (index.project.main.0b86c842259010072178.js:2:1673269)
    at Object.51110 (index.project.main.0b86c842259010072178.js:2:1289198)
    at i (index.project.main.0b86c842259010072178.js:2:1673269)
    at Object.93586 (index.project.main.0b86c842259010072178.js:2:1285860)
    at i (index.project.main.0b86c842259010072178.js:2:1673269)
    at Object.79740 (index.project.main.0b86c842259010072178.js:2:1191813)
    at i (index.project.main.0b86c842259010072178.js:2:1673269)
    at Object.50371 (index.project.main.0b86c842259010072178.js:2:1294230)
    at i (index.project.main.0b86c842259010072178.js:2:1673269)

```

**What you expected to happen**:
业务管理平台界面能正常出现

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- TKE Version:    v1.8.1
- Global or business cluster: Web UI
- Kubernetes version (use `kubectl version`): v1.21.4-tke.1
- Install addons: 
- Others:

**Comment by zouxycn** [comment]
I encoutered the same issue.
And I can't created namespace in "业务管理".It complains :cluster: unknown provider "Baremetal" (forgotten import?).
I am not usre if they are the same root cause.

**Comment by leoryu** [comment]
duplicate https://github.com/tkestack/tke/issues/1634

**Comment by panpan0000** [comment]
@leoryu  I disagreed to close this issue.
(1) the commit https://github.com/tkestack/tke/pull/1641 was not found in release-1.8 branch, and the latest release is 1.8.1. There's no other way to get the latest web ui image release.
(2)  I met the same problem with @zouxycn , there's no prove that above PR fixed this issue as well.

**Comment by leoryu** [comment]
@panpan0000 
version=88f78d334e5961cf6086a131195484c3d2d56c93 && wget https://tke-release-1251707795.cos.ap-guangzhou.myqcloud.com/tke-installer-linux-amd64-$version.run{,.sha256} && sha256sum --check --status tke-installer-linux-amd64-$version.run.sha256 && chmod +x tke-installer-linux-amd64-$version.run && ./tke-installer-linux-amd64-$version.run

please test this daily build version.

this issue will be fixed in 1.9 release, and we plan to release 1.9 this month.
